--- librtas-1.3.13/rules.mk.orig	2015-07-01 21:13:58.128115757 +0200
+++ librtas-1.3.13/rules.mk	2015-07-01 21:24:28.838089289 +0200
@@ -210,7 +210,6 @@
 
 # Default target for building object files
 %.o: %.c
-	@echo "CC $(WORK_DIR)/$@"
-	@$(CC) -c $(CFLAGS) $<
+	$(CC) -c $(CFLAGS) $<
 
 
--- librtas-1.3.13/Makefile.orig	2014-10-23 04:23:41.000000000 +0200
+++ librtas-1.3.13/Makefile	2015-07-01 21:35:00.668062773 +0200
@@ -4,8 +4,6 @@
 
 include rules.mk
 
-.SILENT:
-
 FILES = README COPYRIGHT
 SUBDIRS = librtas_src librtasevent_src libofdt_src
 
