--- librtas-1.3.13/librtasevent_src/print_rtas_event.c.orig	2014-10-23 04:23:41.000000000 +0200
+++ librtas-1.3.13/librtasevent_src/print_rtas_event.c	2015-07-01 21:23:45.884757758 +0200
@@ -330,7 +330,7 @@
 
             if (newline != NULL) {
                 prnt_len = newline - &tmpbuf[offset] + 1;
-                snprintf(buf + buf_offset, prnt_len, &tmpbuf[offset]);
+                snprintf(buf + buf_offset, prnt_len, "%s", &tmpbuf[offset]);
                 buf_offset = strlen(buf);
                 buf_offset += snprintf(buf + buf_offset,
 				       sizeof(buf) - buf_offset, "\n");
@@ -350,7 +350,7 @@
             }
 
             /* print up to the last brkpt */
-            snprintf(buf + buf_offset, prnt_len, &tmpbuf[offset]);
+            snprintf(buf + buf_offset, prnt_len, "%s", &tmpbuf[offset]);
             buf_offset = strlen(buf);
             buf_offset += snprintf(buf + buf_offset, sizeof(buf) - buf_offset,
 				   "\n");
@@ -361,10 +361,10 @@
     } 
 
     prnt_len = snprintf(buf + buf_offset, sizeof(buf) - buf_offset,
-			&tmpbuf[offset]);
+			"%s", &tmpbuf[offset]);
     line_offset += prnt_len;
 
-    return fprintf(ostream, buf);
+    return fprintf(ostream, "%s", buf);
 }
 
 /** 
